src/Tasks/AssemblyDependency/ResolveAssemblyReference.cs

```diff
@@ -1037,7 +1036,9 @@ ArrayList generalResolutionExceptions
 
                                 if (conflictVictims.Length == 0)
                                 {
-                                    foundAtLeastOneUnresolvableConflict = true;
+                                    // This warning is logged regardless of AutoUnify since it means a conflict existed where the reference
+                                    // chosen was not the conflict victor in a version comparison, in other words it was older.
+                                    Log.LogWarningWithCodeFromResources("ResolveAssemblyReference.FoundConflicts", idealRemappingPartialAssemblyName.Name);
```

Comment:
I think you're right.

Noting that `conflictVictims` is also invariant across the `for-j` loop, I've promoted both variable and warning message out of the loop.

I tried to create a unit test that generated multiple identical messages but didn't manage to find a way.